vcsutils/consts.go

```diff
@@ -27,7 +27,9 @@ func (v VcsProvider) String() string {
 type WebhookEvent string
 
 const (
+	PrClosed  WebhookEvent = "PrClosed"
 	PrCreated WebhookEvent = "PrCreated"
```

Comment:
I don't suppose `PrCreated` could be changed to `PrOpened`?

Reply:
Makes sense. I'll change the value.